File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Phys/PHY0001U00GrdRehaSinryouryouCodeInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Phys
{
    [Serializable]
	public class PHY0001U00GrdRehaSinryouryouCodeInfo
	{
		private String _code;
		private String _hangmogCode;
		private String _hangmogName;
		private String _rowState;

		public String Code
		{
			get { return this._code; }
			set { this._code = value; }
		}

		public String HangmogCode
		{
			get { return this._hangmogCode; }
			set { this._hangmogCode = value; }
		}

		public String HangmogName
		{
			get { return this._hangmogName; }
			set { this._hangmogName = value; }
		}

		public String RowState
		{
			get { return this._rowState; }
			set { this._rowState = value; }
		}

		public PHY0001U00GrdRehaSinryouryouCodeInfo() { }

		public PHY0001U00GrdRehaSinryouryouCodeInfo(String code, String hangmogCode, String hangmogName, String rowState)
		{
			this._code = code;
			this._hangmogCode = hangmogCode;
			this._hangmogName = hangmogName;
			this._rowState = rowState;
		}

	}
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/phys/PHY0001U00GrdRehaSinryouryouCodeInfo.java
package nta.med.data.model.ihis.phys;

public class PHY0001U00GrdRehaSinryouryouCodeInfo {
	private String code;
    private String hangmogCode;
    private String hangmogName;
	public PHY0001U00GrdRehaSinryouryouCodeInfo(String code,
			String hangmogCode, String hangmogName) {
		super();
		this.code = code;
		this.hangmogCode = hangmogCode;
		this.hangmogName = hangmogName;
	}
	public String getCode() {
		return code;
	}
	public void setCode(String code) {
		this.code = code;
	}
	public String getHangmogCode() {
		return hangmogCode;
	}
	public void setHangmogCode(String hangmogCode) {
		this.hangmogCode = hangmogCode;
	}
	public String getHangmogName() {
		return hangmogName;
	}
	public void setHangmogName(String hangmogName) {
		this.hangmogName = hangmogName;
	}
}
